project/MiMa.scala

```diff
@@ -1131,6 +1131,11 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.cluster.sharding.ShardRegion.shardBuffers"),
         ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.cluster.sharding.ShardRegion.shardBuffers_="),
 
+        // #18470 Calls to stash() inside of persist callbacks should throw exception
```

Comment:
What should I do? Or Should I wait until 2.6 version? Could you give me some hints?